controller/sync.go

```diff
@@ -241,14 +240,17 @@ func (sc *syncContext) sync() {
 	}
 
 	// any running tasks, lets wait...
-	if tasks.Find(func(t *syncTask) bool { return t.running() }) != nil {
+	if tasks.Any(func(t *syncTask) bool { return t.running() }) {
```

Comment:
A helper function borrowed from Pythons `any()` that I believe is more explicit than `task.Find`